client/commonFramework/output-display.js

```diff
@@ -37,14 +50,16 @@ window.common = (function(global) {
 
   codeOutput.setSize('100%', '100%');
 
-  common.updateOutputDisplay = function updateOutputDisplay(str = '') {
-    codeOutput.setValue(str);
-    return str;
+  common.updateOutputDisplay = function updateOutputDisplay(displayVal = '') {
+    const convertedDisplayVal = convertDisplayValue(displayVal);
+    codeOutput.setValue(convertedDisplayVal);
+    return convertedDisplayVal;
   };
 
-  common.appendToOutputDisplay = function appendToOutputDisplay(str = '') {
-    codeOutput.setValue(codeOutput.getValue() + str);
-    return str;
+  common.appendToOutputDisplay = function appendToOutputDisplay(displayVal = '') {
```

Comment:
This has a minor lint warning about the line being > 80 characters long. @BerkeleyTrue Do these need to be named functions since they're being assigned to a property of the `common` object?